src/editor/index.js

```diff
@@ -359,7 +366,7 @@ class Aganippe {
           eventCenter.dispatch('selectionChange', selectionChanges)
           eventCenter.dispatch('selectionFormats', formats)
           this.dispatchChange()
-        }, 1000)
```

Comment:
If this is the "write" time-out. Why not it set to 100 or 200ms?

Reply:
I just want to delay this handler to the next event loop.